webpack.config.js

```diff
@@ -133,3 +136,16 @@ module.exports = (env, argv = {}) => {
     },
   };
 };
+
+async function getBuildInfo(isDevelopment) {
+  return {
+    isDevelopment,
+    version: (await execOutput("git describe")).trim(),
```

Comment:
i think this should be:
```js
version: isDevelopment ? (await execOutput("git rev-parse HEAD")).trim() : packageData.version,
```

Reply:
If the version building this is a tag, then `git describe` will be just the tag name. The extra info is only included on builds from non-tags. I think it should stay as `git describe`.

Reply:
could we add something that complains loudly if the tag name does not match the version in `package.json`? something akin to:
https://github.com/rehandalal/therapist/blob/master/.circleci/config.yml#L103
https://github.com/rehandalal/therapist/blob/master/setup.py#L21

Reply:
Sure.

Fyi, for this project version *should* (and I realize this won't always happen) using `yarn version --minor` (or `--major` or `--patch`). That updates `package.json` and makes a git tag automatically. It's easy enough to enforce that they match though.